rfc/001-composable-pipelines/README.md

```diff
@@ -118,6 +112,11 @@ data.
 | `load`   | `Void`     | `Arrow`     | Read data from a provided location
 | `store`  | `Arrow`    | `Void`      | Write data into a provided location
 
+These operators take two arguments:
+
+1. **Carrier**: an URL like `s3://aws` or `-` for stdin
+2. **Format**: `json`, `csv`, `feather`, `parquet`, etc.
+
```

Comment:
This was specified further in the document, but is already necessary at this point in the reading.

Reply:
First, I think we don't need to duplicate this information. If we move it up, I would delete it down below and reference it.

Second, I think it doesn't really matter here because we're not providing examples. We're not going into the specification of the operators, so I think it's not too important here. (Not a strong opinion, though.)

Reply:
there is an example right below for which this introduction is useful:

> Another to look at this is that export FORMAT EXPR | op1 | op2 creates two pipelines:
> - `where EXPR | op1 | op2`
> - `store - FORMAT`

I wasn't sure about repeating it, I think you are right.